.prh.yaml

```diff
@@ -0,0 +1,129 @@
+version: 1
+rules:
+  - pattern: vald
+    expected: Vald
+    options:
+      wordBoundary: true
+    specs:
+      - from: vald
+        to: Vald
+      - from: valdcli
+        to: valdcli
+  - pattern: helm
+    expected: Helm
+    options:
+      wordBoundary: true
+  - pattern:
+      - Golang
+      - golang
+      - /go(?! to)/
+    expected: Go
+    options:
+      wordBoundary: true
+    specs:
+      - from: go
+        to: Go
+      - from: going
+        to: going
+      - from: go to
+        to: go to
+      - from: golang
+        to: Go
+      - from: algorithm
+        to: algorithm
+  - pattern: python
+    expected: Python
+    options:
+      wordBoundary: true
+  - pattern: clojure
+    expected: Clojure
+    options:
+      wordBoundary: true
+  - pattern: java
+    expected: Java
+    options:
+      wordBoundary: true
+  - pattern:
+      - node.js
+      - nodejs
+    expected: Node.js
+  - pattern:
+      - Yahoo Japan!
+      - Yahoo! Japan
+      - YJ
+    expected: Yahoo Japan
+  - pattern: ngt
+    expected: NGT
+    options:
+      wordBoundary: true
+    specs:
+      - from: length
+        to: length
+      - from: ngt
+        to: NGT
+  - pattern: ngtd
+    expected: NGTD
+    options:
+      wordBoundary: true
+  - pattern: kubernetes
+    expected: Kubernetes
+    options:
+      wordBoundary: true
+  - pattern: docker
+    expected: Docker
+    options:
+      wordBoundary: true
+  - pattern:
+      - github
+      - Github
+    expected: GitHub
+    options:
+      wordBoundary: true
+  - pattern: ann
+    expected: ANN
+    options:
+      wordBoundary: true
+    specs:
+      - from: planning
+        to: planning
```

Comment:
Same. What does mean? :thinking:

Reply:
@vankichi I think it is because the word `planning` contains `ann` characters, so he added this rule to avoid this wrong suggestion.

Reply:
Ref: 
https://github.com/prh/prh/blob/master/misc/prh.yml#L81

Reply:
@hlts2 
If necessary, maybe it is better to use regular expression on the pattern to avoid this unnecessary rules.

Reply:
@vankichi  @kevindiu This is for test of pattern matching.
we often use it to test when we use regular expression on the pattern.
But in this case, pattern is simple, so we can delete spec field.

Reply:
ah, I got it.
So, we should not remove it.

Reply:
@vankichi
Thank you 🙏 
I have reverted previous modifications.